Decred Wallet/Features/Overview/SyncProgressViewController.swift

```diff
@@ -178,15 +178,15 @@ extension SyncProgressViewController: SyncProgressListenerProtocol {
             return timeFormatter.string(from: TimeInterval(time))!
         }
         
-        var debugSyncInfo = "All Times\n"
-        debugSyncInfo += "elapsed: \(formatTime(debugInfo.totalTimeElapsed))"
-        debugSyncInfo += " remain: \(formatTime(debugInfo.totalTimeRemaining))"
-        debugSyncInfo += " total: \(formatTime(debugInfo.totalTimeElapsed + debugInfo.totalTimeRemaining))\n"
+        var debugSyncInfo = "\(LocalizedStrings.allTimes)\n"
+        debugSyncInfo += "\(LocalizedStrings.elapsed): \(formatTime(debugInfo.totalTimeElapsed))"
+        debugSyncInfo += "\(LocalizedStrings.remain): \(formatTime(debugInfo.totalTimeRemaining))"
```

Comment:
There's a space before the "remain" text, everything is in one line, need to separate the words with space. Same for the "total" text.